### PR TITLE
Create Sanjay Soldering

### PR DESCRIPTION
**Pull request by sanjay-nagesh**
## Describe your project in 1-3 sentences

I will be to constructing a cube-shaped object with a 4x4 array of Light-Emitting Diodes (LEDs). This LED 4x4 cube can be used for various purpose like display, decoration, etc. It will involve soldering and programming to control the LEDs and bring the cube to life.

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

**Comment by sanjay-nagesh** [comment]
slack handle: sanjay

**Comment by bellesea** [comment]
heyy thanks so much for making a PR! we're back on tuesday and will start reviewing PRs in the order that they came on. 

to make the process smoother, please ensure you double checked that your file is properly formatted, you have all the parts you need, and that your plan is detailed enough for us to imagine how you're using the components in your budget to build your project and visualise you work on it :)

and fyi, we're looking for projects where/that:
- you can spend at least 10 days building (from Feb 14th to 23rd)
- you are able to modify or make your own in some ways
- you can learn something new from
- involves working with electronics or software on a lower level than a pre-assembled kit of parts

and, we normally don’t approve projects where you buy consumer components w/o modification like keyboards (buying PCB, keycaps, switches, and assembling it), PCs (buying motherboard, cpu, memory and plugging it together), and kits (eg. closed-sourced consumer drone kits). 

_(note: from a first glance of your project i'm wondering: what's the purpose of the esp8266s and what do you plan to do with the addressable LED strip in your budget?)_

our Jan 15 11:59pm ET deadline is for PRs to come in and not for final approval so we'll work with you to either accept this project OR give you a chance to submit another PR. 

if you have any questions, please do not hesitate to ask her or DM me (i'm belle on slack), thank you!

**Comment by sanjay-nagesh** [comment]
>from a first glance of your project i'm wondering: what's the purpose of the esp8266s and what do you plan to do with the addressable LED strip in your budget? 

The ESP8266s is used for practicing soldering and the addressable LED strip is for soldering project and hanging it up on my room

**Comment by bellesea** [comment]
> > from a first glance of your project i'm wondering: what's the purpose of the esp8266s and what do you plan to do with the addressable LED strip in your budget?
> 
> The ESP8266s is used for practicing soldering and the addressable LED strip is for soldering project and hanging it up on my room

@sanjay-nagesh practicing soldering is SUPER awesome and a great idea but this grant is technically for funding hardware/electronics based projects (to some extent i think your project counts) so it'd be really awesome if we could see a little more of that? like perhaps you could research using the ESP9266s to connect your lights to a web app so you can control it from your phone, allowing you to both practice soldering while building smth :)

**Comment by sanjay-nagesh** [comment]
> > > from a first glance of your project i'm wondering: what's the purpose of the esp8266s and what do you plan to do with the addressable LED strip in your budget?
> > 
> > 
> > The ESP8266s is used for practicing soldering and the addressable LED strip is for soldering project and hanging it up on my room
> 
> @sanjay-nagesh practicing soldering is SUPER awesome and a great idea but this grant is technically for funding hardware/electronics based projects (to some extent i think your project counts) so it'd be really awesome if we could see a little more of that? like perhaps you could research using the ESP9266s to connect your lights to a web app so you can control it from your phone, allowing you to both practice soldering while building smth :)

Hey!

I just changed my project to a LED cube project if that's okay!

I have updated the files accordingly 

Thanks,
Sanjay Nagesh

**Comment by bellesea** [comment]
@sanjay-nagesh awesome!! just to check, are you referencing any guides of sorts and do you plan to use the esp8266 as your microcontroller?

**Comment by sanjay-nagesh** [comment]
> @sanjay-nagesh awesome!! just to check, are you referencing any guides of sorts and do you plan to use the esp8266 as your microcontroller?

I am referencing a guide and yes

**Comment by bellesea** [comment]
cool! can you please add the guide to your file? also, a heads up that we've started reviewing projects but due to the flurry last weekend (100+ PRs ksdjhfkjdshf) it might take a while before we review yours, thank you for your patience!

**Comment by sanjay-nagesh** [comment]
> cool! can you please add the guide to your file? also, a heads up that we've started reviewing projects but due to the flurry last weekend (100+ PRs ksdjhfkjdshf) it might take a while before we review yours, thank you for your patience!

I don't have a specific guide but I might just use https://www.youtube.com/watch?v=K33bRo2tIYg&t=419s

**Comment by sanjay-nagesh** [comment]
I think I will review my whole project and make another final commit because I want to add some parts.

**Review by alhardwarehyde — APPROVED** [review]
Working with LEDs is a lot of fun! I'm also a big fan of soldering so hope you enjoy!

@hackclub/bank-ops for their approval

**Review by hackliv — APPROVED** [review]
@sanjay-nagesh this sounds so cool! I'd love to hear more about your plans for this 😁 dming you now on Slack to chat about next steps for the grant!

File: sanjay-nagesh.md [hunk 1]
```diff
@@ -34,16 +34,17 @@ What materials will you need for your project? Where will you get them? How much
...
+| Soldering Iron  | https://a.co/d/fFcL4K9                | $49.99 |
+| Soldering IRon wire| https://www.amazon.com/dp/B0943RZCMV/ref=twister_B099NZ9GSG?_encoding=UTF8&th=1| $12.39
+|Ardiuno Nano | https://a.co/d/5Th6syh | $24.99
+|PCB Board | https://a.co/d/5Th6syh| $12.99 
+|Header | https://a.co/d/b33iYUa | $12.99
+| LED Lights | https://a.co/d/0u174Qb| $11.99
+| Resistor | https://a.co/d/c4RxLoj | $7.99
+| Wood   | https://a.co/d/bpX8hm9 | $11.95
+| Wire | https://a.co/d/bpX8hm9 | $16.95
+| DOT NFC CARD |https://dotcards.net/checkouts/c/5b2e6084fd441f66c1f4fffbd2dbe0f0/shipping | $44.10 *shipping included* |
```
Comment (bellesea) [inline sanjay-nagesh.md:46]: i think you linked the checkout instead of the product? also, are you referencing any guides for this? thanks!

**Review by bellesea — CHANGES_REQUESTED** [review]

File: sanjay-nagesh.md (hunk 1 above)
Comment (sanjay-nagesh) [reply sanjay-nagesh.md:46]: https://www.youtube.com/watch?v=K33bRo2tIYg&t=419s 
https://dotcards.net/products/dot-bundle

**Review by bellesea — CHANGES_REQUESTED** [review]

**Review by bellesea — APPROVED** [review]

**Pull request merged**